### PR TITLE
mac linux fixes

### PR DESCRIPTION
**Pull request by petersteneteg**

### CODE REVIEW

File: cmake/vcpkghelpers.cmake [hunk 1]
```diff
@@ -76,6 +75,9 @@ function(ivw_vcpkg_paths)
     endif()
 endfunction()
 
+if(VCPKG_TOOLCHAIN)
```
Comment (ResearchDaniel) [inline cmake/vcpkghelpers.cmake]: Why outside any function?

**Review by ResearchDaniel — CHANGES_REQUESTED** [review]
Looks good except some free floating stuff

File: cmake/vcpkghelpers.cmake (hunk 1 above)
Comment (petersteneteg) [reply cmake/vcpkghelpers.cmake]: I only need to do it once, then I can just use the ivw_vcpkg_sha var in the function. Think it gets captured kind of.

Comment (petersteneteg) [reply cmake/vcpkghelpers.cmake]: hence it is not by accident...

Comment (ResearchDaniel) [reply cmake/vcpkghelpers.cmake]: I think that you should put it in the top and state that it is for initialization in that case

**Review by ResearchDaniel — APPROVED** [review]

**Pull request merged**